libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -99,24 +99,26 @@ - (id)contentFromData:(NSData*)data fromResponse:(NSHTTPURLResponse*)response er
     NSError *responseError = nil;
     id content = [super contentFromData:data fromResponse:response error:&responseError];
 
-    if (content && !responseError) {
+    if (!responseError) {
```

Comment:
I wonder some of the code in this method in CSFSalesforceAction belongs in the super class?